addon/experiment-data.json

```diff
@@ -71,7 +71,6 @@
         "avatar": "\/\/www.gravatar.com\/avatar\/abe193c513e181a8a3ee40980928af39?s=64"
       }
     ],
-    "installations_url": "http:\/\/testpilot.dev:8000\/api\/experiments\/1\/installations\/",
```

Comment:
Technically, forward slashes *should* be escaped in JSON, but many serializers don't do it and most parsers don't care.